content/applications/finance/accounting/payables/pay/sepa.rst

```diff
@@ -2,135 +2,96 @@
+**'pain.001.001.03.ch.02'** for Switzerland and **'pain.001.003.03'** for Germany.
 
-.. image:: sepa/sepa01.png
-  :align: center
+Once the payments are processed by your bank, you can directly import the account statement in
+Odoo. The bank reconciliation process will seamlessly match the SEPA orders you sent to your bank
+with actual bank statements.
 
-Pay with SEPA
+Configuration
 =============
 
-Register your payments
-----------------------
-
-You can register a payment that is not related to a supplier bill. To do
-so, use the top menu :menuselection:`Purchases --> Payments`. Register your
-payment and select a payment method by Sepa Credit Transfer.
+Activate the SEPA Credit Transfer (SCT)
+---------------------------------------
 
-If it's the first time you pay this vendor, you will have to fill in the
-Recipient Bank Account field with, at least, the bank name, IBAN and BIC
-(Bank Identifier Code). Odoo will automatically verify the IBAN format.
+To pay suppliers with SEPA, you must activate the **SEPA Credit Transfer** setting.
+To do so, go to :menuselection:`Accounting app --> Configuration --> Settings -->
+Vendor Payments: --> SEPA Credit Transfer (SCT)`. By activating the setting and filling in with your
+company data, you will be able to add the SCT option when paying your vendor.
 
-For future payments to this vendor, Odoo will propose you automatically
-the bank accounts but you will be able to select another one or create a
-new one.
+.. note::
 
-If you pay a specific supplier bill, put the reference of the bill in
-the **memo** field.
+   According to the localization package installed, the **SEPA Direct Debit** and
+   **SEPA Credit Transfer** modules may be installed by default. If not, they need to be installed
+   :doc:`/applications/general/apps_modules`
 
-.. image:: sepa/sepa02.png
-  :align: center
+Activate SEPA payment methods on banks
+--------------------------------------
 
-Once your payment is registered, don't forget to Confirm it. You can
-also pay vendor bills from the bill directly using the Register Payment
-button on top of a vendor bill. The form is the same, but the payment is
-directly linked to the bill and will be automatically reconciled to it.
+From the accounting dashboard, click on the drop-down menu (:guilabel:`⋮`) on your bank journal and
+go to :guilabel:`Configuration`.
+To activate SEPA, click the :guilabel:`Outgoing Payments` tab for outgoing payments. You can add, if
+not already present, SEPA Credit Transfer as a payment method.
 
-Generate SEPA files
--------------------
+Make sure to specify the IBAN account number (domestic account number do not work with SEPA) and the
+BIC (bank identifier code) in the :guilabel:`Journal Entries` tab.
 
-From your accounting dashboard, you should see if there are SEPA files
-to generate for every bank account.
+Pay with SEPA
+=============
 
-.. image:: sepa/sepa03.png
-  :align: center
+Register payments
```

Comment:
Is this section useful?

Reply:
I reduced by erasing the customer payment since we explain the same thing in the SEPA batch payment page

Reply:
I didn't mean that this section is useless, but that having a single sub-section under "Pay with SEPA" is probably unwanted. If it's ok like this, just say so ;)